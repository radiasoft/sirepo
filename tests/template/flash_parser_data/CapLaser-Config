REQUIRES Driver
REQUIRES physics/Hydro/HydroMain/unsplit/MHD_StaggeredMesh

REQUESTS physics/Eos/EosMain/multiTemp/Multitype
REQUESTS physics/Eos/EosMain/Tabulated
REQUESTS physics/Diffuse/DiffuseMain/Unsplit
REQUESTS physics/sourceTerms/EnergyDeposition
REQUESTS physics/sourceTerms/Heatexchange/HeatexchangeMain/Spitzer
REQUESTS physics/materialProperties/Conductivity/ConductivityMain/SpitzerHighZ
REQUESTS physics/materialProperties/MagneticResistivity/MagneticResistivityMain/SpitzerHighZ

# This simulation requires specifying the species via the setup
# line. You must include the species setup variable:
#
# species=fill,wall
#
# An example of a setup line for this simulation is:
#
# ./setup -auto magnetoHD/CapLaserBELLA -site=rsX -2d -nxb=32 -nyb=32 +hdf5typeio species=fill,wall +mtmmmt +usm3t +mgd mgd_meshgroups=6 -parfile=bella_setup.par -objdir=BELLArun +laser ed_maxPulses=1 ed_maxPulseSections=4 ed_maxBeams=1

DATAFILES al-imx-004.cn4
DATAFILES h-imx-004.cn4

##########################
#                        #
#   RUNTIME PARAMETERS   #
#                        #
##########################

D sim_peakField peak field in gauss at capillary wall
PARAMETER sim_peakField   REAL 2.4e3

D sim_period circuit period in seconds
PARAMETER sim_period   REAL 300.e-9

# Target material defaults set for aluminum at room temperature:
D sim_rhoTarg Initial target density
PARAMETER sim_rhoWall   REAL 2.7

D sim_teleTarg Initial target electron temperature
PARAMETER sim_teleWall  REAL 290.11375

D sim_tionTarg Initial target ion temperature
PARAMETER sim_tionWall  REAL 290.11375

D sim_tradTarg Initial target radiation temperature
PARAMETER sim_tradWall  REAL 290.11375

D sim_zminTarg target minimum zbar allowed
PARAMETER sim_zminWall  REAL 0.0

D sim_eosTarg chamber EOS type
PARAMETER sim_eosWall   STRING "eos_tab" ["eos_tab","eos_gam"]

D sim_condWall Wall conductivity at the gas/wall interface
PARAMETER sim_condWall   REAL 1.95e5

# Chamber material defaults set for Helium at pressure 1.6 mbar:
D sim_rhoCham Initial chamber density
PARAMETER sim_rhoFill   REAL 2.655e-07

D sim_teleCham Initial chamber electron temperature
PARAMETER sim_teleFill  REAL 290.11375

D sim_tionCham Initial chamber ion temperature
PARAMETER sim_tionFill  REAL 290.11375

D sim_tradCham Initial chamber radiation temperature
PARAMETER sim_tradFill  REAL 290.11375

D sim_eosCham chamber EOS type
PARAMETER sim_eosFill   STRING "eos_tab" ["eos_tab","eos_gam"]

D cvio_variable Heatexchange saves Cv for ions here, for debugging
VARIABLE CVIO
D cvel_variable Heatexchange saves Cv for electrons here, for debugging
VARIABLE CVEL
VARIABLE KAPA
#VARIABLE RHCD
VARIABLE BDRY

VARIABLE EX
VARIABLE EY
VARIABLE NELE
VARIABLE BRSC
VARIABLE BRMX
VARIABLE ANGL
VARIABLE RESI
