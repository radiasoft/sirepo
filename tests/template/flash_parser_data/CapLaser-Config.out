[
    {
        "_id": 1,
        "_type": "REQUIRES",
        "unit": "Driver"
    },
    {
        "_id": 2,
        "_type": "REQUIRES",
        "unit": "physics/Hydro/HydroMain/unsplit/MHD_StaggeredMesh"
    },
    {
        "_id": 3,
        "_type": "REQUESTS",
        "unit": "physics/Eos/EosMain/multiTemp/Multitype"
    },
    {
        "_id": 4,
        "_type": "REQUESTS",
        "unit": "physics/Eos/EosMain/Tabulated"
    },
    {
        "_id": 5,
        "_type": "REQUESTS",
        "unit": "physics/Diffuse/DiffuseMain/Unsplit"
    },
    {
        "_id": 6,
        "_type": "REQUESTS",
        "unit": "physics/sourceTerms/EnergyDeposition"
    },
    {
        "_id": 7,
        "_type": "REQUESTS",
        "unit": "physics/sourceTerms/Heatexchange/HeatexchangeMain/Spitzer"
    },
    {
        "_id": 8,
        "_type": "REQUESTS",
        "unit": "physics/materialProperties/Conductivity/ConductivityMain/SpitzerHighZ"
    },
    {
        "_id": 9,
        "_type": "REQUESTS",
        "unit": "physics/materialProperties/MagneticResistivity/MagneticResistivityMain/SpitzerHighZ"
    },
    {
        "_id": 10,
        "_type": "DATAFILES",
        "wildcard": "al-imx-004.cn4"
    },
    {
        "_id": 11,
        "_type": "DATAFILES",
        "wildcard": "h-imx-004.cn4"
    },
    {
        "_id": 13,
        "_type": "PARAMETER",
        "comment": "peak field in gauss at capillary wall",
        "default": "2.4e3",
        "isConstant": "0",
        "name": "sim_peakField",
        "range": "",
        "type": "REAL"
    },
    {
        "_id": 15,
        "_type": "PARAMETER",
        "comment": "circuit period in seconds",
        "default": "300.e-9",
        "isConstant": "0",
        "name": "sim_period",
        "range": "",
        "type": "REAL"
    },
    {
        "_id": 17,
        "_type": "PARAMETER",
        "default": "2.7",
        "isConstant": "0",
        "name": "sim_rhoWall",
        "range": "",
        "type": "REAL"
    },
    {
        "_id": 19,
        "_type": "PARAMETER",
        "default": "290.11375",
        "isConstant": "0",
        "name": "sim_teleWall",
        "range": "",
        "type": "REAL"
    },
    {
        "_id": 21,
        "_type": "PARAMETER",
        "default": "290.11375",
        "isConstant": "0",
        "name": "sim_tionWall",
        "range": "",
        "type": "REAL"
    },
    {
        "_id": 23,
        "_type": "PARAMETER",
        "default": "290.11375",
        "isConstant": "0",
        "name": "sim_tradWall",
        "range": "",
        "type": "REAL"
    },
    {
        "_id": 25,
        "_type": "PARAMETER",
        "default": "0.0",
        "isConstant": "0",
        "name": "sim_zminWall",
        "range": "",
        "type": "REAL"
    },
    {
        "_id": 27,
        "_type": "PARAMETER",
        "default": "eos_tab",
        "isConstant": "0",
        "name": "sim_eosWall",
        "range": "\"eos_tab\",\"eos_gam\"",
        "type": "STRING"
    },
    {
        "_id": 29,
        "_type": "PARAMETER",
        "comment": "Wall conductivity at the gas/wall interface",
        "default": "1.95e5",
        "isConstant": "0",
        "name": "sim_condWall",
        "range": "",
        "type": "REAL"
    },
    {
        "_id": 31,
        "_type": "PARAMETER",
        "default": "2.655e-07",
        "isConstant": "0",
        "name": "sim_rhoFill",
        "range": "",
        "type": "REAL"
    },
    {
        "_id": 33,
        "_type": "PARAMETER",
        "default": "290.11375",
        "isConstant": "0",
        "name": "sim_teleFill",
        "range": "",
        "type": "REAL"
    },
    {
        "_id": 35,
        "_type": "PARAMETER",
        "default": "290.11375",
        "isConstant": "0",
        "name": "sim_tionFill",
        "range": "",
        "type": "REAL"
    },
    {
        "_id": 37,
        "_type": "PARAMETER",
        "default": "290.11375",
        "isConstant": "0",
        "name": "sim_tradFill",
        "range": "",
        "type": "REAL"
    },
    {
        "_id": 39,
        "_type": "PARAMETER",
        "default": "eos_tab",
        "isConstant": "0",
        "name": "sim_eosFill",
        "range": "\"eos_tab\",\"eos_gam\"",
        "type": "STRING"
    },
    {
        "_id": 41,
        "_type": "VARIABLE",
        "name": "CVIO",
        "vartype": ""
    },
    {
        "_id": 43,
        "_type": "VARIABLE",
        "name": "CVEL",
        "vartype": ""
    },
    {
        "_id": 44,
        "_type": "VARIABLE",
        "name": "KAPA",
        "vartype": ""
    },
    {
        "_id": 45,
        "_type": "VARIABLE",
        "name": "BDRY",
        "vartype": ""
    },
    {
        "_id": 46,
        "_type": "VARIABLE",
        "name": "EX",
        "vartype": ""
    },
    {
        "_id": 47,
        "_type": "VARIABLE",
        "name": "EY",
        "vartype": ""
    },
    {
        "_id": 48,
        "_type": "VARIABLE",
        "name": "NELE",
        "vartype": ""
    },
    {
        "_id": 49,
        "_type": "VARIABLE",
        "name": "BRSC",
        "vartype": ""
    },
    {
        "_id": 50,
        "_type": "VARIABLE",
        "name": "BRMX",
        "vartype": ""
    },
    {
        "_id": 51,
        "_type": "VARIABLE",
        "name": "ANGL",
        "vartype": ""
    },
    {
        "_id": 52,
        "_type": "VARIABLE",
        "name": "RESI",
        "vartype": ""
    }
]
